GUI/test/javafxgui/javafxcomponents/GUIViewTest.java

```diff
@@ -36,7 +31,7 @@ public void setUp() {
         defaultBoard = new Board(3);
         scene = new Scene(new StackPane(), WINDOW_HEIGHT, WINDOW_WIDTH);
         guiView = new GUIView(scene);
-        controllerStub = new TTTControllerStub(guiView, new GUIGameMaker(new GUIPlayerFactory()));
+        controllerStub = new GUIAppControllerStub(guiView, new GUIGameMaker());
```

Comment:
let's take a look at these tests together. i want to make sure you're testing at the right level here.